Library/Homebrew/os/mac.rb

```diff
@@ -9,9 +9,9 @@
 
 module OS
   module Mac
-    extend self
+    module_function
 
-    ::MacOS = self # compatibility
+    ::Object.const_set(:MacOS, self) # compatibility
```

Comment:
It's probably better to simply exclude this file from the `ConstantName` cop instead of this workaround. Opinions?